scio-bigquery/src/main/scala/com/spotify/scio/bigquery/BigQueryIO.scala

```diff
@@ -279,7 +279,7 @@ final case class BigQueryTypedTable[T: Coder](
 
   override protected def read(sc: ScioContext, params: ReadP): SCollection[T] = {
     val io = reader.from(table.ref).withCoder(CoderMaterializer.beam(sc, Coder[T]))
-    sc.wrap(sc.pipeline.apply(s"Read BQ table ${table.spec}", io))
+    sc.wrap(sc.applyInternal(s"Read BQ table ${table.spec}", io))
```

Comment:
I think eventually we will need to track down all the custom names we have and add this.

Reply:
I didn't find any other usage of this. This was the only place.

Reply:
oh good to know! The good thing about this change is that we can now safely add more default human readable names to transforms! which is a big plus